README.md

```diff
@@ -4,4 +4,149 @@
 + Explain what hoisting is
 + Explain why hoisting is important to remember
 + Avoid variable hoisting
-+ Avoid function hoisting
\ No newline at end of file
++ Avoid function hoisting
```

Comment:
maybe this objective could be "avoid the unintended consequences of function and variable hoisting" ?